test/EventStore.Client.PersistentSubscriptions.Tests/SubscriptionToAll/list_persistent_subscriptions.cs

```diff
@@ -0,0 +1,49 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EventStore.Client.SubscriptionToAll {
+	public class list_persistent_subscriptions : IClassFixture<list_persistent_subscriptions.Fixture> {
+		private readonly Fixture _fixture;
+		private const int AllStreamSubscriptionCount = 3;
+		private const int StreamSubscriptionCount = 4;
+		private const int TotalSubscriptionCount = AllStreamSubscriptionCount + StreamSubscriptionCount;
+		private const string GroupName = nameof(list_persistent_subscriptions);
+		private const string StreamName = nameof(list_persistent_subscriptions);
+
+		public list_persistent_subscriptions(Fixture fixture) {
+			_fixture = fixture;
+		}
+
+		[Fact]
+		public async Task returns_subscriptions_to_all_stream() {
```

Comment:
**SUGGESTION:** I think we should also check if persistent subscriptions to $all methods will throw an expected error for the database version that doesn't support it. The same comment applies for other test for persistent subscription to $all.